homeassistant/components/tado/manifest.json

```diff
@@ -14,5 +14,5 @@
   },
   "iot_class": "cloud_polling",
   "loggers": ["PyTado"],
-  "requirements": ["python-tado==0.17.6"]
+  "requirements": ["python-tado==0.17.7"]
```

Comment:
please bump this is a separate PR

Reply:
That doesn’t really work, as the changes here are dependent on the library bump 😅
Or should we create a PR for that which then needs to be merged first for this to be merged?

Reply:
yes the second 👍. First bump the lib, then this PR

Reply:
Alright, it is in #129842